covid_key/forms.py

```
@@ -0,0 +1,58 @@
+from django import forms
+from django.conf import settings
+from datetime import datetime
+
+from phonenumber_field.formfields import PhoneNumberField
+
+from dependency_injector.wiring import inject, Provide
+from portal.containers import Container
+from portal.services import NotifyService
+from portal.forms import HealthcareBaseForm
+from portal.widgets import CDSRadioWidget
+
+
+class OtkSmsForm(HealthcareBaseForm):
+    """
+    A form to specify patient phone number for receiving one time key (otk) by SMS
+    """
+
+    phone_number = PhoneNumberField(
+        label="Enter patient’s mobile phone number", max_length=100
+    )
+    phone_number2 = PhoneNumberField(
+        label="Confirm patient’s mobile phone number", max_length=100
+    )
+
```

I should have noticed this, but I didn't.

There isn't currently validation to check that both numbers match each other. I tried entering two badly formatted phone numbers, and I got the error I expected, but if you enter 2 different phone numbers, the form submits correctly and sends a message to the first number entered. 

We should return an error message if the two numbers are different and also write a unit test for this.

---

Ya for sure, nice catch